src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +137,46 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+  //get the info object from the server 
+  public ServerInfo getVersion() {
+
```

Comment:
You're right: Consistency with the surrounding code trumps conformance with the style guide.

When this happens at work (finding existing code that is not style-conformant), we generally leave it alone in whatever code review we find it in and then fix it in a followup change.

In any case, this code LGTM as is, then.